damus/Views/ConfigView.swift

```diff
@@ -214,6 +214,16 @@ struct ConfigView: View {
                         .toggleStyle(.switch)
                     Toggle(NSLocalizedString("Zap Vibration", comment: "Setting to enable vibration on zap"), isOn: $settings.zap_vibration)
                         .toggleStyle(.switch)
+                    Toggle(NSLocalizedString("Zap Notification", comment: "Setting to enable Zap Local Notification"), isOn: $settings.zap_notification)
```

Comment:
can we have a new notification setting view instead of putting this all in here

Reply:
@jb55 added the notification toggles in a separate setting view now, screenshot below
![Simulator Screen Shot - iPhone 14 Pro - 2023-04-03 at 14 18 21](https://user-images.githubusercontent.com/120697811/229593554-7c4e565e-4e33-454c-9e7c-016ad0686281.png)

Reply:
@jb55-
![Simulator Screen Shot - iPhone 14 Pro - 2023-04-03 at 19 07 20](https://user-images.githubusercontent.com/120697811/229645706-0c02a663-5e60-4840-90ed-420825497134.png)